index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@300&display=swap'); </style>
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@300&display=swap'); </style>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="mobile-view">
+            <nav class="mobile-header">
+                <header class="mobile-header">
+                    <div class="display-header">
+                        <a href=""><p id="portfolio-name">John Doe </p></a>
+                    </div>
+                    <div class="signal-div">
+                        <div class="union-div">
+                            <img src="images/Union.png" alt="hamburger">
+                        </div>
+                    </div>
+                </header>
+            </nav>
```

Comment:
Great job implementing the changes requested by the previous reviewer. According to HTML semantic tags usage,  the `nav` tag should be a child to the header and not vice-versa, you can refer to this [link ](https://www.w3schools.com/html/html5_semantic_elements.asp)for more explanation.😎⚔️
 Kindly refactor your code to look like this 👇
```
`<header class="mobile-header">
       <div class="display-header">
           <a href=""><p id="portfolio-name">John Doe </p></a>
          </div>
       <nav class="mobile-header">
                  <img src="images/Union.png" alt="hamburger">
          </nav>
      </header>
`
```

[OPTIONAL] I think it will be more professional if you use your name instead of John Doe.☺️

Reply:
@Lizdev-05 Well noted. All changes have been made